client/src/components/Form/Form.scss

```diff
@@ -123,6 +123,8 @@
 
 .form__field-extra-label,
 .form__field-description {
+  display: flex;
```

Comment:
@Cheddam I'm confused about why this was added. It seems to mis-align the content.

Reply:
Can you provide an example of this misalignment? I introduced it (along with `align-items: center;`) in order to fix the alignment, not break it 😅

Reply:
Without flex, align:

![image](https://user-images.githubusercontent.com/541886/83586005-7c2e1780-a59f-11ea-82dc-3690b3122678.png)

With flex, align:

![image](https://user-images.githubusercontent.com/541886/83586022-8a7c3380-a59f-11ea-99df-188b9d487c02.png)

Reply:
Can you check how Firefox vs Chrome behaves?

Reply:
I'm really not sure how you're hitting this @bergice. Here's what it looks like for me in Chrome 83:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/242621/84349326-2b579800-ac0b-11ea-8889-a707cd4bca1b.png">

And this is Firefox 77:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/242621/84349452-783b6e80-ac0b-11ea-8b9b-723a6645920d.png">

Reply:
Okay, me neither. I just tested it again and it looks fine. 🤷